concent_api/conductor/tests/test_integration_conductor.py

```diff
@@ -55,7 +55,7 @@ def _prepare_verification_request_with_blender_subtask_definition(self):
 
         blender_subtask_definition = BlenderSubtaskDefinition(
             verification_request=verification_request,
-            output_format=BlenderSubtaskDefinition.OutputFormat.JPG.name,  # pylint: disable=no-member
+            output_format=BlenderSubtaskDefinition.OutputFormat.JPG,  # pylint: disable=no-member
```

Comment:
Remove `# pylint: disable=no-member` here and below.

Reply:
done